core/controlplane/config/templates/cluster.yaml

```diff
@@ -836,7 +836,15 @@ worker:
 #     # See https://github.com/kubernetes-incubator/kube-aws/issues/230 for more information.
 #     rbac:
 #       enabled: true
-#
+#   # This flag disables automatic security group rule management for ELBs.  
+#   # It requires that the user has setup a rule that allows inbound traffic on kubelet ports 
+#   # from the local VPC subnet so that load balancers can access it.
+#   # Disable setting a new security group rule for every ELB.
+#   disableSecurityGroupIngress: false
```

Comment:
AFAIK, the flag controls whether the k8s aws provider automatically configures every node's security group to include an additional sg ingress rule per k8s managed elb.
Then, how about naming this like `autoConfigureNodeSecurityGroupToAllowTrafficFromELB: true|false` for more clarity?